packages/wonder-blocks-data/src/components/data.ts

```diff
@@ -57,18 +57,18 @@ type Props<
  * support server-side rendering and efficient caching.
  */
 // TODO(FEI-5000): Update this support generic props correctly
-const Data: React.FC<Props<any>> = <TData extends ValidCacheData>({
+const Data: React.FC<Props<any>> = (<TData extends ValidCacheData>({
```

Comment:
**question:** It's a shame we lose the generic support here. Is that going to lead to incorrect usage?

Reply:
I think we can fix this, I just don't know how at the moment. :(